src/main/java/org/opensearch/ad/task/ADTaskManager.java

```diff
@@ -311,13 +311,17 @@ public void startDetector(
         ActionListener<AnomalyDetectorJobResponse> listener
     ) {
         getDetector(detectorId, (detector) -> {
-            if (validateDetector(detector, listener)) { // validate if detector is ready to start
+            if (!detector.isPresent()) {
```

Comment:
Not for this line but general question for this class. Seems just merged code from main branch, no specific code change related with this new model integration, right?

Reply:
The following lines of ADTaskManager are related to new model integration:

old file: 189~212 vs new file: 195~196

old file: 258~268 vs new file: 226~228

old file: 342~362 vs new file: 303~331

Reply:
Don't see difference for line `195~196` in new file.
Are these line numbers for class `ADTaskCacheManager.java` ?

Reply:
![PR](https://user-images.githubusercontent.com/5303417/134603590-dba7427f-4acc-4cca-954a-a49f3d4ef325.png)

Reply:
Are we looking at the same thing?

Reply:
Yes, I have reviewed this class `ADTaskCacheManager.java`.  I'm asking this class `ADTaskManager.java`, what's the specific change for this new model integration.

Reply:
oh.  For ADTaskManager, the related changes are:

old file: 2512~2544; new file: 2530~2562

Reply:
Got it, these changes looks good